libs/ml/include/ml/charge_estimation/ops/constants.hpp

```diff
@@ -33,6 +33,19 @@ static constexpr OperationsCount FLATTEN_PER_ELEMENT             = 1;
 static constexpr OperationsCount MULTIPLICATION_PER_ELEMENT      = 3;
 static constexpr OperationsCount PLACEHOLDER_READING_PER_ELEMENT = 0;
 static constexpr OperationsCount WEIGHTS_READING_PER_ELEMENT     = 0;
+static constexpr OperationsCount LOG_PER_ELEMENT                 = 1;
+static constexpr OperationsCount MAX_PER_ELEMENT                 = 1;
+static constexpr OperationsCount EXP_PER_ELEMENT                 = 1;
+static constexpr OperationsCount DIVISION_PER_ELEMENT            = 1;
+static constexpr OperationsCount CROSS_ENTROPY_PER_ELEMENT =
+    2 * SUBTRACTION_PER_ELEMENT + LOG_PER_ELEMENT;
+static constexpr OperationsCount MEAN_SQ_ERROR_PER_ELEMENT =
```

Comment:
I admire the idea of explaining what does a charge cost of a loss consist of using previously declared charges. Though I guess we need to double-check if these equations are correct.

Reply:
yes, but anyway all of this will need tweaking and adjustments in the next sprint along with benchmarking to verify the precision of the estimates